test/lib/builder/builder.js

```diff
@@ -464,7 +472,30 @@ test("Build application.j", (t) => {
 		// Check for all directories and files
 		assert.directoryDeepEqual(destPath, expectedPath);
 		// Check for all file contents
-		return checkFileContentsIgnoreLineFeeds(expectedFiles, expectedPath, destPath);
+		return checkFileContentsIgnoreLineFeeds(t, expectedFiles, expectedPath, destPath);
+	}).then(() => {
+		t.pass();
+	});
+});
+
+test("Build application.j with resources.json", (t) => {
+	const destPath = "./test/tmp/build/application.j/dest-resources-json";
+	const expectedPath = path.join("test", "expected", "build", "application.j", "dest-resources-json");
+
+	return builder.build({
+		includedTasks: [
+			"generateResourcesJson"
+		],
+		tree: applicationJTree,
+		destPath,
+		excludedTasks: ["createDebugFiles", "generateStandaloneAppBundle", "generateVersionInfo"]
```

Comment:
If we would not exclude `generateVersionInfo` here, this test would reproduce the issue I reported for the openui5-sample-app: https://github.com/SAP/ui5-builder/pull/390#issuecomment-672058203

Reply:
this seems related to the resource path:
"other resources": `/resources/application/j/manifest.json`
"sap-ui-version.json": `/resources/sap-ui-version.json`
--> the prefix `/resources/application/j/` is expected by the grouping logic

Reply:
sap-ui-version.json was excluded with edbf669
It is not relevant for the resources.json and should not be considered.

Reply:
Will other files with a path like `/resources/myfile.xyz` still break the build?

Reply:
yes this currently breaks it.
But this is typically the result of a custom task, "normal" resources should reside in `/resources/application/j` folder